### PR TITLE
more generic renku error message

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

**Comment by burnout87** [comment]
What do you think @volodymyrss ?
image

File: js/mmoda.instrument.js [hunk 1]
```diff
@@ -676,10 +676,11 @@ function panel_title(srcname, param) {
           publish_result_type = 'success';
           if (renku_publish_textStatus == 'error') {
             if (typeof serverResponse === 'object' && serverResponse.hasOwnProperty('error_message'))
-              serverResponse = serverResponse.error_message;
+              serverResponse = `\"${serverResponse.error_message}\" - we will work to address the issue.`;
             else
-              serverResponse = serverResponse;
-            publish_response_title = 'Error while publishing to the Renku repository: ';
+              serverResponse = "we will work to address the issue.";
+            publish_response_title = 'An internal error occured while publishing to Renku:';
```
Comment (volodymyrss) [inline js/mmoda.instrument.js:682]: I suggest shorter

```suggestion
            publish_response_title = 'Could not publish to Renku:';                        
```

Also, could we put the statuspage.io links somewhere?

Comment (burnout87) [reply js/mmoda.instrument.js:682]: I will try to fit them...my concern is just about the space, since I wanted to fit the entire message on one line

Comment (volodymyrss) [reply js/mmoda.instrument.js:682]: They would be urls, so they could even fit in icons?
Text might be further shortened. Yes, space is a concern.

Comment (burnout87) [reply js/mmoda.instrument.js:682]: Much better like this

image

**Review by volodymyrss — APPROVED** [review]

**Pull request merged**